front/components/app/NewBlock.jsx

```diff
@@ -35,6 +35,13 @@ export default function NewBlock({ spec, disabled, onClick, direction }) {
       description:
         "Run a snippet of JavaScript to modify, augment, or combine results from other blocks.",
     },
+    {
+      type: "curl",
+      typeNames: ["curl"],
+      name: "Curl Request",
+      description:
+        "Perform a HTTP request to interface with external services.",
```

Comment:
I'd change "a" to "an" but folks may differ on this usage.

Reply:
Although this site claims that "an HTTP" is correct, it depends I think on whether you are using British or American English. Americans pronounce it "AITCH tee tee pee", so it starts with a vowel sound and takes "an". British folks tend to say "HAITCH tee tee pee" with an "h" sound, so it starts with a consonant and takes "a". So... do what you want, I guess! 🤣

Reply:
:+1: